Libraries/LibGfx/Bitmap.h

```diff
@@ -79,6 +79,13 @@ static StorageFormat determine_storage_format(BitmapFormat format)
     }
 }
 
+struct BackingStore {
+public:
+    void* data;
+    size_t pitch;
+    size_t size_in_bytes;
```

Comment:
Please always initialize primitive variables:
```suggestion
    void* data { nullptr };
    size_t pitch { 0 };
    size_t size_in_bytes { 0 };
```